### ISSUE TITLE
Geojson file with name and description as feature properties

### ISSUE THREAD

**Issue by marissabyrnes** [TerriaJS/neii-viewer#21]
Hello,

Using the NEII viewer http://viewer.neii.gov.au/ to view a geojson file, if a feature has a property named "name" or "description" then the properties won't show.  After renaming these properties to "p1" and "p2" the properties display as expected.

Please find the test files attached.

[test-geojson.zip](https://github.com/TerriaJS/neii-viewer/files/225137/test-geojson.zip)

Thanks,
Marissa

**Comment by stevage** [comment]
Thanks for the report! Just to clarify - the biggest issue here is that none of the other attributes display? (That is, is the lack of the word "name" and "description" itself a problem?)

**Comment by marissabyrnes** [comment]
Yes, once either an attribute called "name" or "description" is present, none of the attributes are displayed.
See attached files:
- example-geojson-neii-viewer.png - when "name" and "description" are renamed to "p1" and "p2" all attributes show
- example-geojson.png - when "name" and "description" are present none of the attributes are shown.

![example-geojson](https://cloud.githubusercontent.com/assets/16189441/14628824/bbff89f6-0644-11e6-8fd2-f3e5d062c963.PNG)
![example-geojson-neii-viewer](https://cloud.githubusercontent.com/assets/16189441/14628826/bda15e4c-0644-11e6-8a9c-937955754774.PNG)

**Comment by marissabyrnes** [comment]
I did some more testing and discovered that it was the property named “description” that was causing the problem.  I just had a quick look at the code and suspect (and I am quite possibly wrong) that the issue is triggered here:

https://github.com/TerriaJS/terriajs/blob/master/lib/ViewModels/FeatureInfoPanelViewModel.js - line 175

    if (defined(this.feature.description)) {
        this.rawData = this.feature.description.getValue(this.terria.clock.currentTime);
    } else if (defined(data)) {
        // There is no template, and no description - just return the properties as JSON.
        this.rawData = JSON.stringify(data);
    }

**Comment by kring** [comment]
@marissabyrnes that's intended behavior, though there's some discussion about changing it.  See TerriaJS/terriajs#1452.  Because changing it would be a breaking change for other applications, we have to approach it with some caution.  How painful is it for you to use a different property name in your GeoJSON files?

**Comment by marissabyrnes** [comment]
@kring thanks for the update.  It is a bit painful for us to make changes to the GeoJSON files, but we need to send out the NEII Viewer link to our data providers to test their GeoJSON files for a June 30 release of the NEMSR product we are building.  In the interest of being able to do that ASAP we will update the GeoJSON files.
